source/common/http/http2/codec_impl.h

```diff
@@ -44,6 +44,7 @@ class Http2CodecImplTestFixture;
 // This is not the full client magic, but it's the smallest size that should be able to
 // differentiate between HTTP/1 and HTTP/2.
 const std::string CLIENT_MAGIC_PREFIX = "PRI * HTTP/2";
+constexpr uint64_t H2_FRAME_HEADER_SIZE = 9;
```

Comment:
This constant already exists in the cc file, please unify them:
`static const uint64_t FRAME_HEADER_SIZE = 9;`

Reply:
Fixed.